ImageList map_images
ImageList live_images
---
Histogram[] histograms